package com.examen.badwallet_api.transaction.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Positive;

import java.math.BigDecimal;

public record DepositRequest(
    @NotNull @Positive BigDecimal amount,
    @NotBlank String paymentMethod
) {}
